### PR TITLE
Rb 1442

### PR DESCRIPTION
**Pull request by LawrieR**

### CODE REVIEW

File: thirdparty/SemanticUI-Angular/src/sm-core.js [hunk 1]
```diff
@@ -325,7 +325,7 @@
 
 			template: [
 				'<div class="menu">',
-				'  <div class="item" ng-repeat="item in items" data-value="{{ getValue(item) }}" sm-html="label({item:item})"></div>',
+				'  <div class="item" ng-class="{\'disabled\': getDisabled(item)}" ng-repeat="item in items" data-value="{{ getValue(item) }}" sm-html="label({item:item})"></div>',
```
Comment (craig-rainbird) [inline thirdparty/SemanticUI-Angular/src/sm-core.js]: Was this auto generated? We won't lose the change if it is updated?

Comment (craig-rainbird) [reply thirdparty/SemanticUI-Angular/src/sm-core.js]: Same question for all of the third party stuff?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ facing release notes.
 
 ## v2.18.0
 
+  *  [Fix]   RB-1441: Add date picker icon.
+  *  [Fix]   RB-1442: Fix date format
+  *  [Fix]   RB-1429/30: Fix Evidence tree icon missing in try-query/agent
```
Comment (craig-rainbird) [inline CHANGELOG.md]: What were the changes for this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ facing release notes.
...
+  *  [Fix]   RB-1432: Fix API Log not scrolling
```
Comment (craig-rainbird) [inline CHANGELOG.md]: What were the changes for this?

File: thirdparty/SemanticUI-Angular/src/sm-core.js (hunk 1 above)
Comment (LawrieR) [reply thirdparty/SemanticUI-Angular/src/sm-core.js]: It's not auto generated but we do need to make edits to that pool of third party work. We do the same thing in rainbird-editor for forked 3rd party repos.

Comment (craig-rainbird) [reply thirdparty/SemanticUI-Angular/src/sm-core.js]: ok no problem

**Pull request merged**